idx/open-in-idx-template/idx-template.json

```diff
@@ -0,0 +1,35 @@
+{
+    "name": "GMP Android Open in IDX",
```

Comment:
"name": "Open in IDX for Maps SDK for Android Samples"

Reply:
Done.